### DISCUSSION TITLE
Precipitation missing data using the python library

### DISCUSSION THREAD

**Discussion by m4zette** [meteostat/meteostat-python#90] [General]
Hi,
First of all thank you for this wonderful library.
However, I encountered a small problem: for some locations (for example Trignac in France on December 24, 2020) I get missing values in the precipitation column using the python library while all the data is available in the web application.

Is it a bug or am I misunderstanding something ?

![données api python](https://user-images.githubusercontent.com/50274273/159017141-c86fbed4-a372-4e31-81e0-3b326a61d6a3.PNG)

![api web](https://user-images.githubusercontent.com/50274273/159019081-aa8fd79f-22a8-4577-931c-0c3a14e40d98.PNG)

Thanks!

**Comment by clampr** [comment]
Hi, Thanks 🙂 

Do you mind sharing your code?

**Reply by m4zette** [reply]
Yes I've made this:

```
# Import Meteostat library and dependencies
from datetime import datetime
from meteostat import Point, Hourly

# Set time period
start = datetime(2020, 12, 24)
end = datetime(2020, 12, 25)

# Create Point Trignac
point = Point(47.3178, -2.1892, 6)

# Get hourly data
data = Hourly(point, start, end).fetch()
```

**Reply by clampr** [reply]
The Meteostat JSON API uses a different `Point.radius`. You can try increasing the radius, e.g:

```py
Point.radius = 120000
```

**Reply by m4zette** [reply]
Great that works perfectly.
Thanks for the reactive support ;)
